sdk/src/main/java/io/opentelemetry/sdk/resources/Resource.java

```diff
@@ -117,6 +117,19 @@ public static Resource create(Attributes attributes) {
     return new AutoValue_Resource(attributes);
   }
 
+  /**
+   * Returns a {@link Resource}. This resource information is loaded from the
+   * OTEL_RESOURCE_ATTRIBUTES environment variable or otel.resource.attributes system properties.
+   *
+   * @return a {@code Resource}.
+   */
+  public static Resource createDefault() {
```

Comment:
I'd suggest creating this value only once, for simplicity and to keep the invariant `EnvAutodetectResource` has - also, for that case, rename it to `getDefault()` too.

Reply:
Thanks. Fixed at lastest commit.